modules/core/resources/META-INF/smithy/formats.smithy

```diff
@@ -0,0 +1,10 @@
+$version: "2"
+
+namespace alloy
+
+/// This trait indicates that a String value contains a date without
+/// a time component.
+/// e.g.: "2022-03-30"
```

Comment:
should this refer to some specific name from ISO8601?

Reply:
sure

Reply:
will fix

Reply:
Updated with info from RFC 3339: https://www.rfc-editor.org/rfc/rfc3339#section-5.6